modules/contacts/modules.php

```diff
@@ -85,6 +85,31 @@ public function process() {
     }
 }
 
+/**
+ * @subpackage contacts/handler
+ */
+class Hm_Handler_process_export_contacts extends Hm_Handler_Module {
+    public function process() {
+        if (array_key_exists('contact_source', $this->request->get)) {
+            $source = $this->request->get['contact_source'];
+            $contact_list = $this->user_config->get('contacts', array());
```

Comment:
Please get the contacts from the contact source, not the user config.

Reply:
Changes applied. Thanks for the review